homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/ProfileScreen.kt

```diff
@@ -115,6 +125,36 @@ internal fun ProfileScreen(uiState: ProfileUiState, onEvent: (ProfileEvent) -> U
     }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun PullToRefreshBox(
+    enabled: Boolean,
+    isRefreshing: Boolean,
+    onRefresh: () -> Unit,
+    modifier: Modifier = Modifier,
+    content: @Composable BoxScope.() -> Unit
+) {
```

Comment:
The default component doesn't have an `enabled` property, so I had to create this.